README.md

```diff
@@ -151,108 +153,107 @@ To find the unique identifier of an extension:
+1. Find the ID of your Azure Subscription [in the Azure portal](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBlade?WT.mc_id=academic-77460-alfredodeza) or [by following this guide](https://learn.microsoft.com/azure/azure-portal/get-subscription-tenant-id?WT.mc_id=academic-77460-alfredodeza).
+2. Create a Service Principal with a "contributor" role that is allowed to make changes to any resources in that subscription. Replace $AZURE_SUBSCRIPTION_ID with the ID you found in step 1 and run this command:
 
 ```
 az ad sp create-for-rbac --name "CICD" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID --sdk-auth
 ``` 
 
-Capture the output and add it as a [repository secret](/../../settings/secrets/actions/new) with the name `AZURE_CREDENTIALS`
+3. Capture the output and add it as a [Github repository secret](/../../settings/secrets/actions/new) with the name `AZURE_CREDENTIALS`.
 
 ### Update workflow file
 
-Now that you have everything created, we need to check the name for our webapp.
+Now that you have all the Azure resources created, you need to let your code repository know your Azure app name.
 
-- Check the result of `randomIdentifier` in your bash shell in az, use that to change in our webapp
-- Update the [.github/workflows/web_app.yml](/../../edit/main/.github/workflows/web_app.yml) file
-- Update this: `AZURE_WEBAPP_NAME`: demo-fastapi-(randomIdentifier-result-here)
+1. Find your app name. It should look something like `demo-fastapi-97709018` but with a different random number at the end,
+and you can find it in the Azure portal or the Cloud Shell commands.
+2. Open the [.github/workflows/web_app.yml](/../../edit/main/.github/workflows/web_app.yml) file and update the value of `AZURE_WEBAPP_NAME` to your app name.
 
 ### Deployment
 
 Before continuing, check the following:
 
-1. You've created an Azure Service Principal and saved it as a [repository secret](/../../settings/secrets/) as `AZURE_CREDENTIALS`
-1. You've created an [App Service](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Web%2Fsites) with a valid name and the site is already available with the default static content
+1. You've created an Azure Service Principal and saved it as a [repository secret](/../../settings/secrets/) as `AZURE_CREDENTIALS`.
+1. You've created an [App Service](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Web%2Fsites) with a valid name and the site is already available with the default static content.
 
 To deploy:
 
 1. Go to [repository actions](/../../actions/workflows/web_app.yml) and click on _Run workflow_ and then the green button to run it.
 
-**Deploying can take a couple of minutes**. Make sure you tail the logs in the Azure cloud shell to check the progress:
+**Deploying can take a couple of minutes**. Make sure you stream the logs in the Azure Cloud Shell to check the progress:
```

Comment:
Changed tail to stream because most students dont know about tail'ing, but not a big deal.